meson_options.txt

```diff
@@ -1 +1,2 @@
 option('man-pages', type: 'feature', value: 'auto', description: 'Generate and install man pages')
+option('varlink', type: 'feature', value: 'auto', description: 'Enable remote control with varlink')
```

Comment:
Maybe rename this option to `ipc`? Whether varlink is used under the hood shouldn't really matter to the users.